weasyl/moderation.py

```diff
@@ -304,10 +307,35 @@ def finduser(userid, form):
             lo.c.email.op('~')(form.email),
             lo.c.email.op('ilike')('%%%s%%' % form.email),
         ))
-    else:
+
+    # Filter for banned and/or suspended accounts
+    if form.excludeactive == "on":
+        q = q.where(d.sa.or_(
+            lo.c.settings.op('~')('b'),
+            lo.c.settings.op('~')('s'),
+        ))
+    if form.excludebanned == "on":
+        q = q.where(lo.c.settings.op('!~')('b'))
+    if form.excludesuspended == "on":
+        q = q.where(lo.c.settings.op('!~')('s'))
+
+    # Filter for date-time
+    if form.dateafter and form.datebefore:
+        q = q.where(d.sa.between(pr.c.unixtime, arrow.get(form.dateafter), arrow.get(form.datebefore)))
+    elif form.dateafter:
+        q = q.where(pr.c.unixtime.op('>=')(arrow.get(form.dateafter)))
```

Comment:
`>=` is available as a Python operator: `q.where(pr.c.unixtime >= arrow.get(form.dateafter))`